plugins/nginx-vhosts/functions

```diff
@@ -114,6 +114,10 @@ validate_nginx() {
 
 restart_nginx() {
   case "$DOKKU_DISTRO" in
+    debian)
+      /usr/sbin/invoke-rc.d nginx reload > /dev/null
+      ;;
+
```

Comment:
I asked in #debian on freenode and they said this was the way to do it, and that `invoke-rc.d` would do the right thing, regardless of the init system in use.